src/actions.ts

```diff
@@ -116,16 +130,9 @@ export async function executeActions(
           },
           "please ensure this environmental variable is not set because graphile-migrate sets it dynamically for children.",
         ),
-        encoding: "utf8",
-        // 50MB of log data should be enough for any reasonable migration... right?
-        maxBuffer: 50 * 1024 * 1024,
+        shell: true,
+        stdio: "inherit",
       });
-      if (stdout) {
-        parsedSettings.logger.info(stdout);
-      }
-      if (stderr) {
-        parsedSettings.logger.error(stderr);
-      }
```

Comment:
So now rather than feeding out through the logger we just write to stdout/stderr? This feels problematic - people using custom loggers to log to external log providers will no longer receive the output from these commands.

Actually... why weren't you receiving the stdout/stderr from your previous script - shouldn't it have been output here?

Reply:
Ah, it isn't output here because an error is thrown 👍

Reply:
It looks like all we needed to do was to catch the error thrown from `exec` and log it's stdout/stderr too before throwing the error again.

> If this method is invoked as its [util.promisify()](https://nodejs.org/api/util.html#utilpromisifyoriginal)ed version, it returns a Promise for an Object with stdout and stderr properties. The returned ChildProcess instance is attached to the Promise as a child property. In case of an error (including any error resulting in an exit code other than 0), a rejected promise is returned, with the same error object given in the callback, but with two additional properties stdout and stderr.
> -- https://nodejs.org/api/child_process.html#child_processexeccommand-options-callback:~:text=execute%20the%20command.-,If%20this%20method%20is%20invoked%20as%20its%20util.promisify()ed,the%20callback%2C%20but%20with%20two%20additional%20properties%20stdout%20and%20stderr.,-const%20util%20%3D